USERS-GUIDE.md

```diff
@@ -594,6 +594,24 @@ possible to specify the bean name using `id` attribute (by default fully qualifi
     class="com.your.organization.configuration.package.ConnectionConfiguration"/>
 ```
 
+`<conf4j:converter/>` registers in the context a type converter.
+This tag expects one attribute `class` which specifies fully qualified name of the converter class. It is also
+possible to specify the bean name using `id` attribute (by default fully qualified class name is used).
+
+```xml
+<conf4j:converter
+    class="com.your.organization.configuration.package.CustomConverter"/>
+```
+
+`<conf4j:delegatingConverterFactory/>` registers in the context a delegating converter factory.
```

Comment:
It deserves better description, and again the name of the element is wrong - it should be `converter-decorator`.